spec/rubocop/cop/lint/parentheses_as_grouped_expression_spec.rb

```diff
@@ -24,6 +24,28 @@
     RUBY
   end
 
+  it 'registers an offense and corrects for method call with space before the parenthesis with block argument' do
+    expect_offense(<<~RUBY)
```

Comment:
This cop is not aimed at the rule of adding parentheses. In other words, this case is a false positive and it should be `expect_no_offenses`, not `expect_offense`.

Reply:
Please add a test case for Ruby 2.7's numbered parameter as well.

Reply:
In this case, the block argument is in parentheses, so there are no new parentheses to be added.
Therefore, this one should remain in offense. How about it?

Reply:
> Please add a test case for Ruby 2.7's numbered parameter as well.

Indeed it is. I added test cases.

Reply:
Ah, this case is correct using `expect_offense` 👍